app/src/main/res/values/strings.xml

```diff
@@ -1,58 +1,58 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
 
-    <string name="app_name">Seedvault</string>
+    <string name="app_name" translatable="false">Seedvault</string>
     <string name="backup">Backup</string>
 
     <string name="restore_backup_button">Restore backup</string>
 
     <!-- Settings -->
     <string name="settings_backup">Backup my data</string>
-    <string name="settings_backup_location">Backup location</string>
-    <string name="settings_backup_location_picker">Choose backup location</string>
-    <string name="settings_backup_location_title">Backup Location</string>
-    <string name="settings_backup_location_invalid">The chosen location can not be used.</string>
+    <string name="settings_backup_location">Backup folder</string>
```

Comment:
We call this location because you don't actually get to select a particular folder to store the backup, just the location.

Reply:
How about "Store backups in"?
Not knowing the distinction between "Backup" and "Back up" could otherwise cause confusion

Reply:
I'm not sure I understand what you mean by "Backup" and "Back up" here.

Reply:
Backup, noun. Back up, verb.